### PR TITLE
Fix atom47 keyboard revision naming

### PR DESCRIPTION
**Pull request by Maartenwut**

### CODE REVIEW

File: keyboards/atom47/rev2/info.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "keyboard_name": "Atom47", 
-  "url": "", 
-  "maintainer": "qmk", 
-  "bootloader": "", 
+  "url": "https://atomkb.eu/atom47", 
+  "maintainer": "Maarten Dekkers", 
+  "bootloader": "qmk-dfu",
```
Comment (mechmerlin) [inline keyboards/atom47/rev2/info.json]: we can remove this as the bootloader tag is no longer used.

File: keyboards/atom47/keymaps/default/keymap.c [hunk 2]
```diff
@@ -13,25 +13,25 @@
 #define _______ KC_TRNS
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-[_MA] = LAYOUT_ansi(
+[_MA] = LAYOUT(
```
Comment (drashna) [inline keyboards/atom47/keymaps/default/keymap.c]: @mechmerlin did we want these to be `LAYOUT` or `LAYOUT_ansi`?

Comment (mechmerlin) [reply keyboards/atom47/keymaps/default/keymap.c]: This is a drop-in replacement for the vortex core, which does not come in ISO variants. 

Based on https://atomkb.eu/atom47/ this board also does not support ISO variants, so there really is no need to say that it is LAYOUT_ansi.

Comment (drashna) [reply keyboards/atom47/keymaps/default/keymap.c]: Okay, just wanted to make sure. :)

**Comment by drashna** [comment]
Thanks!

**Pull request merged**